addons/website/models/ir_ui_view.py

```diff
@@ -157,7 +146,7 @@ def unlink(self):
 
         specific_views = self.env['ir.ui.view']
         if self and self.pool._init:
-            for view in self:
+            for view in self.filtered(lambda view: not view.website_id):
```

Comment:
@JKE-be this fixes the last migration issue I had. on init if a website specific view was deleted (can happen on write) it will try to delete the other specific views. That makes no sense functionally plus the other views can still be referenced as inherited thus throwing constraint error when being unlinked.